content/help/ops/setup/app-health-check/index.md

```diff
@@ -171,6 +171,25 @@ NAME                             READY     STATUS    RESTARTS   AGE
 liveness-http-975595bb6-5b2z7c   2/2       Running   0           1m
 {{< /text >}}
 
+#### Probe rewrite cleanup
+
+Remove the service and redeploy Istio with Helm option `sidecarInjectorWebhook.rewriteAppHTTPProbe=false`.
+
+1. To remove the service, run:
+
+{{< text bash >}}
+$ kubectl delete -f <(istioctl kube-inject -f @samples/health-check/liveness-command.yaml@)
+{{< /text >}}
+
+1. To reinstall Istio, run:
```

Comment:
Isn't that description too vague? I worry readers might ask themselves: "When did I uninstall it?" and get confused.